client/blocks/importer/wordpress/import-everything/pre-migration/credentials.tsx

```diff
@@ -38,6 +38,17 @@ export function Credentials( props: Props ) {
 		<div className="import__pre-migration import__import-everything import__import-everything--redesign">
 			<div className="import__heading import__heading-center">
 				<Title>{ translate( 'You are ready to migrate' ) }</Title>
+				<SubTitle className="onboarding-subtitle--full-width">
+					{ translate(
+						'Provide your SSH server credentials to migrate %(sourceSite)s to %(targetSite)s',
```

Comment:
Better to add some comments for the translator to explain what the args is here.

Reply:
Agree; added!